ush/calcanl_gfs.py

```diff
@@ -12,10 +12,83 @@
 from collections import OrderedDict
 import datetime
 from wxflow import cast_as_dtype
+from netCDF4 import Dataset, num2date
+import numpy as np
 
 python2fortran_bool = {True: '.true.', False: '.false.'}
 
 
+def calcanl_gcafs(DoIAU, l4DEnsVar, Write4Danl, ComOut, APrefix,
```

Comment:
just noting that we might want to do something slightly different here since the original script contains code that is not very pretty and nearing the end of its life

Reply:
@aerorahul are you okay if we merge this in as-is but I create an issue to address this technical debt in 2026?

Reply:
@ypwang19 I recall you said that we couldn't use the Fortran code for calc_anl because the background mass values for the aerosols are not correct. Is that correct? I think this implementation needs to be reworked a bit. We should be able to leverage the calc_anl infrastructure from GDAS in GCDAS as-is for the meteorology and then add the aerosol increments as an additional step. Does that make sense?